src/tests/class_tests/openms/source/IsotopeDistribution_test.cpp

```diff
@@ -170,6 +170,18 @@ START_SECTION(Size getMin() const)
 	TEST_EQUAL(iso4.getMin(), 48)
 END_SECTION
 
+START_SECTION(Size getMostAbundant() const)
+	IsotopeDistribution iso(EmpiricalFormula("C1").getIsotopeDistribution(CoarseIsotopePatternGenerator(11, true)));
+    // The most abundant isotope is the monoisotope
+	TEST_EQUAL(iso.getMostAbundant().getMZ(), 12)
+	IsotopeDistribution iso2(EmpiricalFormula("C100").getIsotopeDistribution(CoarseIsotopePatternGenerator(11, true)));
+    // In this case, the most abundant isotope isn't the monoisotope
+	TEST_EQUAL(iso2.getMostAbundant().getMZ(), 1201)
+	IsotopeDistribution iso3;
+    // Making sure an empty distribution doesn't crash it.
+    TEST_EQUAL(iso3.getMostAbundant().getMZ(), 0);
```

Comment:
this *should* not work, `std::max_element` will return `last()` on which you cannot call `getMZ()`

Reply:
Ah good point. Any thoughts on what I should return if it's empty? Peak1D of 0mz and 1 probability? 0 and 0?

Reply:
either that or throw an exception that you cannot get a max peak on an empty distribution - which will be better than a segfault down the line :-)

Reply:
> I see now that the constructor initializes the distribution to have a single entry of 0,1. No wonder it didn't crash.

what happens if someone initializes it on purpose with 0 entries?

Reply:
I added a check for that. If the distribution is empty, it will return 0,1